packages/devtools_app/lib/src/shared/vm_service_wrapper.dart

```diff
@@ -1166,15 +1166,6 @@ class VmServiceWrapper implements VmService {
     }
   }
 
-  /// Gets the name of the service stream for the connected VM service. Pre-v3.22
```

Comment:
I wonder if we can start dropping these version checks now that we're shipping via the SDK... Might be worth getting rid of special cases like there everywhere at some point.

Reply:
Agreed. Filed https://github.com/flutter/devtools/issues/3699 to track.